src/pages/disclaimer.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Layout from '../components/layout';
+import ContentContainer from '../components/content-container';
+
+const Disclaimer = () => {
+  const heading = 'Legal Disclaimer';
+
+  const content = (
+    <p>
+      Graasp Insights (the &lsquo;software&rsquo;) and its source code are
```

Comment:
Not sure if these should also go in `t()` tags. If it's too complex, let's make a different Issue and address it separately (if French is needed, for instance).